docs/core-protocol/public-lock/README.md

```diff
@@ -64,6 +22,85 @@ The Lock Smart Contract has multiple capabilities:
 
 ## Upgrades and customization
 
-All locks deployed prior to version 10 (to be released by year end 2022) are NOT upgradable, which means their core logic will remain unchanged. Starting with version 10, locks are upgradable by their lock manager, through the Unlock contract.
+All locks deployed prior to version 10 are NOT upgradable, which means their core logic will remain unchanged. Starting with version 10, locks are upgradable by their lock manager, through the Unlock contract.
 
 Lock managers can also alter the behavior of their locks thru the use of [hooks](./hooks/).
+
+## Changelog
+
+### Version 11
+
+**Released**: August 2022
+
+This new version brings a lot of gas savings as well as a few minor bug fixes. It also introduces the following features:
+
+### `referrerFees` and `setReferrerFee`
+
+This is the most important addition to the protocol in this version. `referrerFees` are ways for a lock manager to share their income with the `referrer` when a purchase is made (when calling `purchase`, `renewMembershipFor`, or `extend`). The fee is set as a percentage in basis points (`1000` is 10%).
+
+A lock manager can call `setReferrerFee` to set specific fees for different referers. It is possible to set a _default_ fee for _any_ referrer by passing the Zero address as the referrer when calling `setReferrerFee`.
+
+The function `referrerFees(address)` can be used to retrieve the fees set for a specific address.
+
+#### `lendKey` and `unlendKey`
+
+These 2 functions can be used by a key manager to lend a key or recover it. They can only be called by the key manager or current owner of the key if no manager is set. `lendKey(from, recipient, tokenId)` will set the caller as the new key manager (see [roles](./access-control.md)), and will transfer ownership of the key to the `recipient` address. `unlendKey(recipient, tokenId)` will transfer the key to the recipient.
```

Comment:
@julien51 "unLendKey transfers the key to the recipient" => confusing. doesn't the "recipient" already have the key if it has been lent to them?